### DISCUSSION TITLE
<u></u> tag displaying

### DISCUSSION THREAD

**Discussion by ihor-zinchenko** [remarkjs/remark#1215] [Q&A] [answered]
Hello! 
Can't find any info how to solve the problem with <u></u> tag. it displays like 
<img width="215" alt="image" src="https://github.com/remarkjs/remark/assets/38561554/929fd5ee-ca25-488e-ac1d-b0811da7b359">
i know that i can replace <u> for example like like this `replace(/<u>(.*?)<\/u>/g, '__$1__')`
but maybe there is a better solution?

Here is my config
```
<Markdown
    components={{
      u: ({ node, ...props }) => {
        console.log(node);
        console.log(props);
        return (<React.Fragment {...props} />);
      },
    }}
    remarkPlugins={[remarkGfm]}
>
    {text}
</Markdown>```

i tried to do something with u tag by using component but seems like its never calls. 
Any ideas?

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @ihor-zinchenko! 👋 
I'm assuming your `<Markdown>` element is `react-markdown`?
If so, please see the readme section on HTML in Markdown https://github.com/remarkjs/react-markdown#appendix-a-html-in-markdown
Why this is happening and how to make the tag render are explained.

**Reply by ihor-zinchenko** [reply]
yes its react-markdown `import ReactMarkdown from "react-markdown";`
but yes, it works with `rehypeRaw`!!! 
Many thanks!!!!!!!!
